examples/kafka/adobe-s3-connector/kafka-blueprint.yaml

```diff
@@ -0,0 +1,220 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: kafka-blueprint
+  namespace: kasten-io
+actions:
+  backup:
+    type: Deployment
+    outputArtifacts:
+      s3Dump:
+        keyValue:
+          s3path: '{{ .Phases.setupPhase.Output.s3path }}'
+          backupDetail: '{{ .Phases.setupPhase.Output.backupDetail }}'
+    phases:
+    - func: KubeTask
+      name: setupPhase
+      args:
+        namespace: "{{ .Object.metadata.namespace }}"
+        podOverride:
+          containers:
+          - name: container
+            imagePullPolicy: Always
+          restartPolicy: Never
+        image: akankshakumari393/adobe-s3-sink-connector:latest
```

Comment:
We use goreleaser to build and push tools images. e.g https://github.com/kanisterio/kanister/blob/master/.goreleaser.yml#L102
Please add new entries in `dockers` sections for building images for Kafka example. You can build the images locally with `goreleaser release --skip-publish --snapshot --rm-dist`